inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/config/CommonConfigHolder.java

```diff
@@ -668,6 +680,15 @@ private void preReadAuditSetting() {
                 this.auditProxys.add(tmpIPPort.trim());
             }
         }
+        // check auditProxys configure
+        if (this.enableAudit) {
+            if (!this.enableAuditProxysDiscoveryFromManager && this.auditProxys.isEmpty()) {
+                LOG.error("{}'s {} must be configured when {} is true and {} is false, exist!",
+                        COMMON_CONFIG_FILE_NAME, KEY_AUDIT_PROXYS, KEY_ENABLE_AUDIT,
+                        KEY_AUDIT_PROXYS_DISCOVERY_MANAGER_ENABLE);
+                System.exit(2);
```

Comment:
why use System.exit not throw a runtime exception ?

Reply:
When the necessary configuration for the correct operation of the system is not met, the error log is directly output and the system exits, so that maintenance personnel can know the cause of the problem earlier and more accurately and solve it.